src/cnd.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RLANG_CND_H
+#define RLANG_CND_H
+
+#include <stdbool.h>
+
+
+void r_inform(const char* fmt, ...);
+void r_warn(const char* fmt, ...);
+void r_abort(const char* fmt, ...);
+
+SEXP interp_str(const char* fmt, ...);
```

Comment:
In the sense that it will clutter the code: `rlang_interp_str()`, `rlang_string()`, `rlang_vec_alloc()` etc, that hinders readability.